frameworks/hdfs/universe/config.json

```diff
@@ -1423,8 +1414,12 @@
         },
         "security_auth_to_local": {
           "type": "string",
-          "description": "Maps kerberos principals to local user names",
-          "default": ""
+          "description": "Maps kerberos principals to local user names. This field must be base64 encoded.",
```

Comment:
Is there another alternative other than base64 encoding? Something like the JSON placement constraints?

Reply:
Not that I'm aware of. This is consistent with how Elastic achieves this functionality.